packages/react-server-dom-webpack/npm/client.node.unbundled.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
+var n, w;
 if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./cjs/react-server-dom-webpack-client.node.unbundled.production.js');
+  n = require('./cjs/react-server-dom-webpack-client.node.unbundled.production.js');
+  w = require('./cjs/react-server-dom-webpack-client.node-webstreams.unbundled.production.js');
 } else {
-  module.exports = require('./cjs/react-server-dom-webpack-client.node.unbundled.development.js');
+  n = require('./cjs/react-server-dom-webpack-client.node.unbundled.development.js');
+  w = require('./cjs/react-server-dom-webpack-client.node-webstreams.unbundled.development.js');
 }
+
+exports.registerServerReference = function (r, i, e) {
+  return w.registerServerReference(n.registerServerReference(r, i, e), i, e);
+};
+exports.createServerReference = function (i, c, e, d, f) {
+  return w.registerServerReference(
+    n.createServerReference(i, c, e, d, f),
+    i,
+    e
+  );
+};
```

Comment:
These ones are stateful registers so we need to register with both bundles since each has its own map.